src/tidy.zig

```diff
@@ -98,7 +98,6 @@ const DeadDetector = struct {
     }
 
     fn deinit(detector: *DeadDetector) void {
-        assert(detector.files.count() == 0); // Sanity-check that `.finish` was called.
```

Comment:
Forgot to why this: this creates extra nosie if there are other tidy errors. ideally, `deinit` would take `?anyerror` and assert only if there's an error, but there's no appropriate form of `defer` for this pattern.